app/services/post_owner_changer.rb

```diff
@@ -23,6 +24,13 @@ def change_owner!
       post.set_owner(@new_owner, @acting_user, @skip_revision)
       PostAction.remove_act(@new_owner, post, PostActionType.types[:like])
 
+      if post.post_number == 1
+        TopicUser.change(old_owner.id, @topic.id, notification_level: 2) if old_owner
+        TopicUser.change(@new_owner.id, @topic.id, notification_level: 3)
```

Comment:
I'd prefer if you would use `NotificationLevels.topic_levels[:watching]` instead of a magic number. Same goes for the other notification levels.